src/python/pants/backend/shell/util_rules/shell_command_test.py

```diff
@@ -617,20 +618,9 @@ def test_shell_command_boot_script(rule_runner: RuleRunner) -> None:
     assert res.argv[1] == "-c"
     assert res.argv[2].startswith("cd src &&")
     assert "bash -c" in res.argv[2]
-    assert res.argv[2].endswith(
-        shlex.quote(
-            "$mkdir -p .bin;"
-            "for tool in $TOOLS; do $ln -sf ${!tool} .bin; done;"
-            'export PATH="$PWD/.bin";'
-            "./command.script"
-        )
-        + " src:boot-script-test"
-    )
+    assert res.argv[2].endswith(shlex.quote("./command.script") + " src:boot-script-test")
 
-    tools = sorted({"python3_8", "mkdir", "ln"})
-    assert sorted(res.env["TOOLS"].split()) == tools
-    for tool in tools:
-        assert res.env[tool].endswith(f"/{tool.replace('_', '.')}")
+    assert "PATH" in res.env
```

Comment:
I'd like to preserve the test for the path to `python3.8` here.. (i.e. assert that there is a `python3.8` on the PATH provided.)